fn/either.go

```diff
@@ -52,19 +52,19 @@ func (e Either[L, R]) IsRight() bool {
 	return e.isRight
 }
 
-// LeftToOption converts a Left value to an Option, returning None if the inner
+// MuteRight converts a Left value to an Option, returning None if the inner
 // Either value is a Right value.
-func (e Either[L, R]) LeftToOption() Option[L] {
+func (e Either[L, R]) MuteRight() Option[L] {
 	if e.isRight {
 		return None[L]()
 	}
 
 	return Some(e.left)
 }
 
-// RightToOption converts a Right value to an Option, returning None if the
+// MuteLeft converts a Right value to an Option, returning None if the
 // inner Either value is a Left value.
-func (e Either[L, R]) RightToOption() Option[R] {
+func (e Either[L, R]) MuteLeft() Option[R] {
```

Comment:
The old names `RightToOption`/`LeftToOption` are more intuitive imo.

Reply:
If a second opinion agrees, I'll change it.

Reply:
+1 for old names `RightToOption` and `LeftToOption`